### DISCUSSION TITLE
UI messed up with latest concourse 7.5.0?

### DISCUSSION THREAD

**Discussion by HemangLavana** [concourse/concourse#7607] [Help & Support] [answered]
![image](https://user-images.githubusercontent.com/516502/135144271-5e761597-84f8-4502-ab7d-4291c5cfb41e.png)

After upgrading to concourse 7.5.0 version, the UI seems messed up in chrome and firefox. I tried clearing the cookies and reloading but it didn't help. Any idea if something is not setup correctly on my end.

**Comment by taylorsilva** [comment, marked as answer]
What versions of Firefox and Chrome? Both are rendering fine for me right now looking at a few pipelines on https://ci.concourse-ci.org/
Is there any pipeline on our CI instance that this happens for you?

Also just opened #7605 for Safari 15 though.

**Reply by HemangLavana** [reply]
Chrome: Version 93.0.4577.82
Firefox: 92.0 (64-bit)
MacOS Big Sur: version 11.6

My colleague was also seeing the same issue. I downgraded to concourse 7.4.0 and the UI is back to normal:

![image](https://user-images.githubusercontent.com/516502/135146344-fef66d52-5895-4083-9183-0404895015b6.png)

All of your pipelines look fine to me. Something in my pipelines must be causing this.

**Reply by HemangLavana** [reply]
Looking at the release notes, I suppose https://github.com/concourse/concourse/pull/7438 may have caused the issue?

**Reply by HemangLavana** [reply]
Even a simple pipeline such as this:

![image](https://user-images.githubusercontent.com/516502/135151941-bd32da04-3773-4354-99b0-2e94a9ad2c9b.png)

gets messed up in 7.5.0:

![image](https://user-images.githubusercontent.com/516502/135152004-0b035bf3-31e6-4b15-909b-6226fc3d47be.png)

Do you think it could be due to use of dot in resource/job names (ex: "set.self") ?

**Reply by taylorsilva** [reply]
Yup, it's the dots. That PR is probably the cause.
<img width="1468" alt="image" src="https://user-images.githubusercontent.com/4583837/135155283-bc1dcf4b-1675-48d5-a95c-1c21f23fae89.png">
```yaml
resources:
- name: a.resource.with.dots
  type: mock

jobs:
- name: job
  plan:
  - get: a.resource.with.dots
```

**Reply by taylorsilva** [reply]
Opened an issue #7608
